common/filestore_utils.py

```diff
@@ -36,24 +36,28 @@ def _using_gsutil():
 
 # TODO(zhichengcai): Change all implementations of cp, ls, and rm to stop using
 # special handling of *args. This is error prone now that there are wrappers.
-def cp(*cp_arguments, **kwargs):  # pylint: disable=invalid-name
-    """Copy source to destination."""
-    return filestore_utils_impl.cp(*cp_arguments, **kwargs)
+def cp(source, destination, recursive=False, parallel=False):  # pylint: disable=invalid-name
+    """Copies |source| to |destination|."""
+    return filestore_utils_impl.cp(source,
+                                   destination,
+                                   recursive=recursive,
+                                   parallel=parallel)
 
 
-def ls(*ls_arguments, must_exist=True, **kwargs):  # pylint: disable=invalid-name
-    """List files or folders."""
-    return filestore_utils_impl.ls(*ls_arguments,
-                                   must_exist=must_exist,
-                                   **kwargs)
+def ls(path, must_exist=True):  # pylint: disable=invalid-name
+    """Lists files or folders in |path|. Doesn't except on failure if must_exist
+    is False otherwise can raise subprocess.CalledProcessError.
+    Returns the output of ls split line-by-line (for example ls('directory/')
+    returns every file in the directory."""
```

Comment:
unclear from the sentence whether a list of strings is returned or a string with one file per line